easybuild/easyblocks/g/gcc.py

```diff
@@ -525,7 +525,7 @@ def sanity_check_step(self):
             bin_files.append('gfortran')
             lib_files.extend(['libgfortran.%s' % sharedlib_ext, 'libgfortran.a'])
 
-        if 'lto' in self.cfg['languages']:
+        if self.cfg['withlto']:
```

Comment:
OK, you've clearly figured this out really well, my apologies for questioning your patch!

Thanks a lot for the very detailed explanation, excellent stuff! o/